### ISSUE TITLE
Duplicate bullets appears on line series

### ISSUE THREAD

**Issue by rafee11** [amcharts/amcharts5#865]
![image](https://user-images.githubusercontent.com/23078836/227500358-50a1c310-b4db-4663-ada9-28a87c7ecd0c.png)
As i have attached the screenshot, i am getting extra bullets without tooltip. It looks like it is adding dummy bullets and even when i move the y axis scale with zoom level it remains same on that position in chart

```
series.bullets.push((root: am5.Root, series: any, dataItem: { dataContext: { transaction_data: { type: string; }; }; }) => {
            if ("condition"){                
                return am5.Bullet.new(root, {
                    sprite: am5.Circle.new(root, {
                        radius: 6,
                        fill: am5.color(0x39FF74),
                        stroke: am5.color(0xffffff),
                        strokeWidth: 0
                    })
                });
            }
            else {
                return am5.Bullet.new(root, {
                    sprite: am5.Circle.new(root, {
                        radius: 6,
                        fill: am5.color(0xFF7439),
                        stroke: am5.color(0xffffff),
                        strokeWidth: 0
                    })
                });
            }        
            
            var tooltip = am5.Tooltip.new(this.root, {
            getFillFromSprite: false,
            getStrokeFromSprite: false,
            autoTextColor: false,
            getLabelFillFromSprite: false,
            labelText: "default text"
        });
        // set tooltip text 
        tooltip.adapters.add("labelText", function<T extends object>(text, target ) {
        
        tooltip.get("background").adapters.add("fill", function<T extends object>(fill, target) {
            if("condition"){
                return am5.color(0x39FF74) ; 
            }
            else {
                return am5.color(0xFF7439);
            }  
        });
        
         series.set("tooltip", tooltip);
```

**Comment by martynasma** [comment]
Would you be able to post your whole chart and data on CodePen/jsFiddle so that we can test?

**Comment by rafee11** [comment]
@martynasma  I am working on the Codepen, i will share it as soon as possible.

**Comment by rafee11** [comment]
@martynasma Here is the codepen sample [](https://codepen.io/rafee11/pen/WNaNBGY) with the logics and sample data format which i have in my application. I am not able to reproduce the duplicate bullets issue here. 

I am trying to get the same issues on codepen

**Comment by rafee11** [comment]
@amcharts  @martynasma I have attached the video reference of duplicate bullets. As we can see the bullets appears at first time without tooltip on fixed position. Even if we move the chart positions it will be appearing on same position. When we zoom in or zoom out using mouse scroll it disappears. 

https://user-images.githubusercontent.com/23078836/230653095-fdde1c7a-4d3b-46c8-a75a-077f14d67dc8.mp4

**Comment by martynasma** [comment]
Can you doublecheck if you are indeed using the latest version in your app? Also, do you get any errors in console?

**Comment by rafee11** [comment]
@martynasma  I am not getting any errors in console. Yes I am using the latest version

**Comment by rafee11** [comment]
@martynasma  I have create a sample app with my data and codes to reproduce the issue. Please check on this repository and let me know. Link https://gitlab.com/rafi-optisol/stock-chart

Run the application as mentioned in read me file, Search 'amazon' on input box and the chart will be appearing on the browser.

**Comment by rafee11** [comment]
It will be appearing like this 
![image](https://user-images.githubusercontent.com/23078836/231011475-60ad43b9-bf3c-426b-9dbd-89407de6c5c7.png)

**Comment by zeroin** [comment]
Do you use setIndex method to add new data for your series?

**Comment by rafee11** [comment]
@zeroin  No, I don't use setIndex method to add new data. I can able to reproduce this with different browsers like Chrome and Mozilla Firefox.

**Comment by rafee11** [comment]
@zeroin  I could not able to reproduce it on codeopen, I have already attached codepen link here https://github.com/amcharts/amcharts5/issues/865#issuecomment-1500123771. 

Can you able to run my public repo and reproduce the issue on your local?

**Comment by martynasma** [comment]
Unfortunately, debugging custom setups is beyond the scope of what we do here in free support.

Do you have a support subscription by any chance? If you do, please reach out to us directly at contact@amcharts.com.

Thank you for understanding!
